react-components/src/components/RuleBasedOutputs/hooks/useExtractUniqueAssetIdsFromMapped.tsx

```diff
@@ -3,13 +3,15 @@ import { type ModelWithAssetMappings } from '../../../hooks/cad/modelWithAssetMa
 import { useMemo } from 'react';
 import { uniqBy } from 'lodash';
 import { isDefined } from '../../../utilities/isDefined';
+import { isClassicCadAssetMapping } from '../../CacheProvider/cad/assetMappingTypes';
 
 export const useExtractUniqueAssetIdsFromMapped = (
   assetMappings: ModelWithAssetMappings[] | undefined
 ): InternalId[] => {
   return useMemo(() => {
     const mappings = assetMappings?.map((item) => item.assetMappings).flat() ?? [];
     const assetIds: InternalId[] = mappings
+      .filter(isClassicCadAssetMapping)
```

Comment:
Should we consider to rename this function / file to `useExtractUniqueClassicAssetIdsFromMapped`